### PR TITLE
Make compatible with modules

### PR DESCRIPTION
**Pull request by xingrz**
Please publish it to NPM if possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+From NPM:
+
+```
+npm install --save webadb
+```
+
+Or, include directly (not recommended):
+
+```html
+<script src="https://github.com/webadb/webadb.js/raw/master/webadb.js"></script>
```
Comment (kidonng) [inline README.md]: Consider using jsDelivr or UNPKG link?

**Comment by cavokz** [comment]
I'm struggling with the ESLint thing, it seems to bring in too much stuff. Is it really all needed?

**Comment by xingrz** [comment]
@kidonng @cavokz updated

**Comment by cavokz** [comment]
Thanks!

**Comment by xingrz** [comment]
@cavokz would you like to publish it NPM?

**Comment by cavokz** [comment]
@xingrz I'm hesitating because of the public email address thing, I already receive enough spam. Is it NPM such an enabler? Does node.js support WebUSB?

**Comment by xingrz** [comment]
@cavokz Not for node.js. NPM is becoming a good practice for web developers.

**Pull request merged**